Makefile

```diff
@@ -37,6 +37,11 @@ start: ## Start everything fresh
 	@docker build -t $(container_tag) .
 	@echo "Creating network..."
 	@docker network create $(network_name) 2>/dev/null || true
+	@echo "Starting Redis..."
+	@docker run -d --rm \
+		--name $(redis_name) \
+		--network $(network_name) \
+		redis:alpine
```

Comment:
is there any good reason why there is no `compose.yml`? 
using test containers to not colide with having compose for running everything, the make file just do what compose does, but is worse - you cannot create healthcheck for redis and start application 

for multiple instances it's possible to use `--scale`